File: sdk/approute-sdk/golang/enum/field_error_code.go
package enum

// FieldErrorCode represents a machine-readable validation error code
// returned in the "errors[].code" field of an API response.
type FieldErrorCode string

const (
	FieldErrorMissing       FieldErrorCode = "MISSING"
	FieldErrorOutOfRange    FieldErrorCode = "OUT_OF_RANGE"
	FieldErrorInvalidFormat FieldErrorCode = "INVALID_FORMAT"
	FieldErrorInvalidValue  FieldErrorCode = "INVALID_VALUE"
	FieldErrorNotAllowed    FieldErrorCode = "NOT_ALLOWED"
	FieldErrorTooLong       FieldErrorCode = "TOO_LONG"
	FieldErrorTooShort      FieldErrorCode = "TOO_SHORT"
)

File: sdk/approute-sdk/javascript/src/enums/field-error-code.ts
/**
 * Field-level error codes returned in validation errors.
 */
export const FieldErrorCode = {
  MISSING: "MISSING",
  OUT_OF_RANGE: "OUT_OF_RANGE",
  INVALID_FORMAT: "INVALID_FORMAT",
  INVALID_VALUE: "INVALID_VALUE",
  NOT_ALLOWED: "NOT_ALLOWED",
  TOO_LONG: "TOO_LONG",
  TOO_SHORT: "TOO_SHORT",
} as const;

export type FieldErrorCode =
  (typeof FieldErrorCode)[keyof typeof FieldErrorCode];
